classes/actions/ActionScheduler_Action.php

```diff
@@ -93,4 +106,30 @@ public function get_group() {
 	public function is_finished() {
 		return FALSE;
 	}
+
+	/**
+	 * Sets the priority of the action.
+	 *
+	 * @param int $priority Priority level (lower is higher priority). Should be in the range 0-255.
+	 *
+	 * @return void
+	 */
+	public function set_priority( $priority ) {
+		if ( $priority < 0 ) {
+			$priority = 0;
+		} elseif ( $priority > 255 ) {
+			$priority = 255;
+		}
+
+		$this->priority = $priority;
```

Comment:
probably cast to `int` here as well

Reply:
Good catch, addressed in https://github.com/woocommerce/action-scheduler/pull/921/commits/fbffde233c0157215d47370ed89c01035a47b4e5.